projects/incubator/src/eventsystem/events/event.emitter.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable, Type } from '@angular/core';
+import { BehaviorSubject, combineLatest, merge, Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { CxEventSource } from './event.model';
+
+/**
+ * The EventEmitter provides a service to control events in Spartacus.
+ *
+ * Events are driven by an Event Type. Event types are global Classes
+ * which might not be in the final build, this depends on the modules
+ * you use.
+ *
+ * Event sources can be attach and dispatch with the `EventEmiter`.
+ * The various sources are combined into a single emission, so that a subscriber
+ * will get the events in a combined observable.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class EventEmitter {
```

Comment:
I personally don't like that to much. If we're forced to make unique names because of potential clashes, i'm willing to give in, but in this case it's a matter of where to import from.

Reply:
I realise by having a 2nd look that even in this branch I did use the cx prefix for some occasions where it wasn't' needed. I'll make up my mind up on that.